import sys


def error(msg: str) -> None:
    print("ERROR:", msg)
    sys.exit(1)
